topics/proteomics/metadata.yaml

```diff
@@ -103,7 +108,7 @@ material:
         github_username: "bgruening"
```

Comment:
The contributors should be like this now: 

```
contributors:
      - timothygriffin
      - pratikdjagtap
      - jj-umn
      - blankclemens
```
With a list of GitHub username.
Can you change it?

Reply:
Done